app/src/main/java/in/krharsh17/barview_sample/MainActivity.java

```diff
@@ -70,11 +70,14 @@ protected void onCreate(Bundle savedInstanceState) {
         ));
 
 
-        barView.setData(barModels);
+
+        barView.setAnimationType(1);
```

Comment:
I had asked for a constant to be defined to be used here. Like the one you have for toasts, `Toast.LENGTH_SHORT` or `Toast.LENGTH_LONG`. A developer would never know 1 stands for what kind of animation.